content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -79,6 +79,8 @@ as an introduction to various technologies and serves as a jumping-off point.
 The following networking options are sorted alphabetically - the order does not
 imply any preferential status.
 
+{{% 3rdparty-content %}}
```

Comment:
I suggest renaming this shortcode (do not include a number in the name).
third-party-content or external-content

Reply:
It would be a good idea to first create the shortcode and add a description in the style guide (or another relevant page) for when to use this shortcode.
Then, create a new PR updating pages with this shortcode.

Reply:
I think that's a good idea too re: a second PR. I'll remove the shortcode from this PR and open a subsequent one.